/**:
    ros__parameters:
        localization_type: "AMCL"